frontend/apps/marketing/src/components/actionBlocks/helpers.ts

```diff
@@ -0,0 +1,21 @@
+// Shows a "New" tag on Action Blocks with content that was published in the last 3 months.
+export const showNewTag = (publishedDate: string) => {
```

Comment:
nit: I think we might make `publishedDate?: string`, so that when calling this function we won't need to do `tag={showNewTag(publishedDate) ? 'New' : undefined}` instead of `publishedDate && showNewTag(publishedDate) ? 'New' : undefined`?

Reply:
I made the changes from [these comments](https://github.com/code-dot-org/code-dot-org/pull/65619/files#r2077715776), but am getting type errors (_Type 'undefined' is not assignable to type 'string'._) when updating this to `tag={showNewTag(publishedDate) ? 'New' : undefined}` so I think it needs to stay as-is.